homeassistant/components/airthings/sensor.py

```diff
@@ -72,6 +78,40 @@
         native_unit_of_measurement=CONCENTRATION_PARTS_PER_BILLION,
         name="VOC",
     ),
+    "light": SensorEntityDescription(
+        key="light",
+        device_class=DEVICE_CLASS_ILLUMINANCE,
+        native_unit_of_measurement=PERCENTAGE,
+        name="Light",
+    ),
+    "virusRisk": SensorEntityDescription(
+        key="virusRisk",
+        native_unit_of_measurement="/ 10",
+        name="Virus Risk",
+    ),
+    "mold": SensorEntityDescription(
+        key="mold",
+        native_unit_of_measurement="/ 10",
+        name="Mold",
+    ),
+    "rssi": SensorEntityDescription(
+        key="rssi",
+        native_unit_of_measurement=SIGNAL_STRENGTH_DECIBELS_MILLIWATT,
```

Comment:
I thought it was SIGNAL_STRENGTH_DECIBELS ?

Reply:
You are right. Fixed.